networking_oneview/ml2/drivers/oneview/common.py

```diff
@@ -347,3 +350,193 @@ def is_rack_server(server_hardware):
     :return: True or False;
     """
     return False if server_hardware.get('locationUri') else True
+
+
+def check_oneview_entities_availability(oneview_client, server_hardware):
```

Comment:
Instead of passing `oneview_client`, we could have within the method: `oneview_client = get_oneview_client()`